serveradmin/resources/management/commands/clear_migration_log.py

```diff
@@ -0,0 +1,49 @@
+"""Serveradmin - igvm_migration_log cleanup
+
+Copyright (c) 2020 InnoGames GmbH
+"""
+from django.core.management.base import BaseCommand
+from adminapi.dataset import Query
+from adminapi.filters import Not, Empty
+from datetime import datetime
+
+class Command(BaseCommand):
+    help = 'Check for old igvm_migration_log entries and deletes them'
+
+    def handle(self, *args, **options):
+        """Entry point for command
+
+        get all hypervisors which have an entry in igvm_migration_log
+        and delete all entries which are from the previous day.
+
+        """
+
+        all_hypervisor = Query({'igvm_migration_log': Not(Empty())},
+                           ['igvm_migration_log', 'hostname'])
+
+        for hypervisor in all_hypervisor:
+
+            for entry in hypervisor['igvm_migration_log']:
+                timestamp = entry.split(' ')[0]
+
+                dt_object_sa = datetime.fromtimestamp(float(timestamp))
```

Comment:
integer is sufficient here and makes more sense as we also store the values as integer in the attribute value.